rclip/model.py

```diff
@@ -37,8 +38,15 @@ def compute_text_features(self, text: str) -> np.ndarray:
 
     return text_encoded.cpu().numpy()
 
-  def compute_similarities_to_text(self, item_features: np.ndarray, text: str) -> List[Tuple[float, int]]:
-    text_features = self.compute_text_features(text)
+  def compute_similarities_to_text(
+      self, item_features: np.ndarray,
+      positive_queries: List[str], negative_queries: List[str]) -> List[Tuple[float, int]]:
+
+    positive_features: List[np.ndarray] = [self.compute_text_features(query) for query in positive_queries]
+    negative_features: List[np.ndarray] = [-self.compute_text_features(query) for query in negative_queries]
+    all_features: List[np.ndarray] = positive_features + negative_features
```

Comment:
Also, why would we store query features instead of computing the "final" features right away?

Like this:
```
text_features = np.zeros((self.VECTOR_SIZE,))
for query in positive_queries:
  text_features += self.compute_text_features(query)
for query in negative_queries:
  text_features -= self.compute_text_features(query)
```

I also believe that we should be able to modify `compute_text_features` to accept multiple queries at once, and batch them, let's say, by eight queries. This should result in a noticeable performance boost when querying via multiple queries. What do you think?